run.py

```diff
@@ -2,41 +2,35 @@
 import asyncio
 import time
 import argparse
-from major_tom.major_tom import MajorTom
+from majortom_gateway import GatewayAPI
 from demo.demo_sat import DemoSat
 
 logger = logging.getLogger(__name__)
 
 # Set up command line arguments
 parser = argparse.ArgumentParser()
-parser._action_groups.pop()
-required = parser.add_argument_group('required arguments')
-optional = parser.add_argument_group('optional arguments')
-required.add_argument(
-    '-m',
-    '--majortomhost',
-    help='Major Tom host name. Can also be an IP address for local development/on prem deployments.',
-    required=True)
-required.add_argument(
-    '-g',
-    '--gatewaytoken',
-    help='Gateway Token used to authenticate the connection. Look this up in Major Tom under the gateway page for the gateway you are trying to connect.',
-    required=True)
-optional.add_argument(
+# Required Args
+parser.add_argument(
+    "majortomhost",
+    help='Major Tom host name. Can also be an IP address for local development/on prem deployments.')
+parser.add_argument(
+    "gatewaytoken",
+    help='Gateway Token used to authenticate the connection. Look this up in Major Tom under the gateway page for the gateway you are trying to connect.')
+
+# Optional Args and Flags
+parser.add_argument(
     '-b',
     '--basicauth',
-    help='Basic Authentication credentials. Not required unless BasicAuth is active on the Major Tom instance. Must be in the format "username:password".',
-    required=False)
-optional.add_argument(
+    help='Basic Authentication credentials. Not required unless BasicAuth is active on the Major Tom instance. Must be in the format "username:password".')
+parser.add_argument(
     '-l',
     '--loglevel',
-    help='Log level for the logger. Defaults to "debug", can be set to "info", or "error".',
-    required=False)
-optional.add_argument(
-    '-t',
-    '--telemetry',
-    help='If included, starts the telemetry beacon for the demo spacecraft automatically in the mode indicated. Can be "nominal" or "error".',
-    required=False)
+    choices=["info", "error"],
```

Comment:
Probably should allow explicit selection of `debug` too?

Reply:
I'd rather leave it as is. It's more accurate to how the behavior is governed.